runtime/vm/createramclass.cpp

```diff
@@ -1595,7 +1595,7 @@ loadSuperClassAndInterfaces(J9VMThread *vmThread, J9ClassLoader *classLoader, J9
 						 */
 						bool interfaceIsPublic = J9_ARE_ALL_BITS_SET(interfaceClass->romClass->modifiers, J9_JAVA_PUBLIC);
 						if ((!interfaceIsPublic && (packageID != interfaceClass->packageID))
-						|| (interfaceIsPublic && !checkModuleAccess(vmThread, vm, romClass, module, interfaceClass->romClass, interfaceClass->module, interfaceClass->packageID, 0))
+						|| (interfaceIsPublic && (J9_VISIBILITY_ALLOWED != checkModuleAccess(vmThread, vm, romClass, module, interfaceClass->romClass, interfaceClass->module, interfaceClass->packageID, 0)))
```

Comment:
Same here, pls add an indent for this line.

In addition, do we expect `checkVisibility` used outside `vm` like `checkVisibility`?
If not, there is no need for `extern J9_CFUNC IDATA  checkModuleAccess(J9VMThread *currentThread, J9JavaVM* vm, J9ROMClass* srcRomClass, J9Module* srcModule, J9ROMClass* destRomClass, J9Module* destModule, UDATA destPackageID, UDATA lookupOptions);` within `j9protos.h`.